runtime/dancebox/src/lib.rs

```diff
@@ -966,6 +972,52 @@ impl pallet_pooled_staking::Config for Runtime {
     type WeightInfo = pallet_pooled_staking::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+    pub ParachainBondAccount: AccountId32 = PalletId(*b"ParaBond").into_account_truncating();
+    pub PendingRewardsAccount: AccountId32 = PalletId(*b"PENDREWD").into_account_truncating();
+    // 5%/year with 2_629_800 blocks per year -> 0.05/2_629_800 = 19/1_000_000_000
+    pub const InflationRate: Perbill = prod_or_fast!(Perbill::from_parts(19), Perbill::from_percent(1));
```

Comment:
@librelois what was the reason for choosing a different inflation rate for dev mode?